sw/airborne/modules/sensors/airspeed_ms45xx_i2c.c

```diff
@@ -136,28 +162,31 @@ struct AirspeedMs45xx ms45xx;
 static struct i2c_transaction ms45xx_trans;
 static Butterworth2LowPass ms45xx_filter;
 
-
 static void ms45xx_downlink(struct transport_tx *trans, struct link_device *dev)
 {
-  pprz_msg_send_AIRSPEED_MS45XX(trans, dev, AC_ID,
-                                &ms45xx.diff_pressure,
-                                &ms45xx.temperature, &ms45xx.airspeed);
+  pprz_msg_send_AIRSPEED_MS45XX(trans,dev,AC_ID,
+                                &ms45xx.pressure,
+                                &ms45xx.temperature,
+                                &ms45xx.airspeed);
 }
 
 void ms45xx_i2c_init(void)
 {
-  ms45xx.diff_pressure = 0;
+  ms45xx.pressure = 0.;
   ms45xx.temperature = 0;
   ms45xx.airspeed = 0.;
+  ms45xx.pressure_type = MS45XX_PRESSURE_TYPE;
   ms45xx.pressure_scale = MS45XX_PRESSURE_SCALE;
   ms45xx.pressure_offset = MS45XX_PRESSURE_OFFSET;
   ms45xx.airspeed_scale = MS45XX_AIRSPEED_SCALE;
   ms45xx.sync_send = MS45XX_SYNC_SEND;
 
   ms45xx_trans.status = I2CTransDone;
   // setup low pass filter with time constant and 100Hz sampling freq
+  //#ifdef USE_AIRSPEED_LOWPASS_FILTER
```

Comment:
please remove if not used

Reply:
Yes still unsure if we should be ableto define it or not. Will change